README.md

```diff
@@ -1,76 +1,76 @@
-| BMSNAV    | &check;              | &check;               | &check;      | &check;       |
-| BMSND     | &cross;              | &cross;               | Missing      | Missing       |
-| BMSSA     | &cross;              | &cross;               | Missing      | Missing       |
-| BMSSD     | &cross;              | &cross;               | &check;      | &check;       |
-| BMSSH     | Missing              | Missing               | &check;      | &check;       |
-| BMSSK     | Missing              | Missing               | &check;      | &check;       |
-| BMSSS     | Missing              | Missing               | &check;      | &check;       |
-| BMSSTOC   | Missing              | Missing               | &cross;      | &cross;       |
-| BMTRE     | &cross;              | &cross;               | &check;      | &check;       |
-| BMTUN     | &check;              | &check;               | Missing      | Missing       |
-| BNVIB     | Missing              | Missing               | &check;      | &check;       |
-| BPSI      | &check;              | &check;               | &check;      | &check;       |
-| BPTDAT    | Missing              | Missing               | &check;      | &check;       |
-| BPTDEF    | Missing              | Missing               | &check;      | &check;       |
-| BREM      | Missing              | Missing               | &check;      | &check;       |
-| BRES      | Missing              | Missing               | &check;      | &check;       |
-| BREV      | Missing              | Missing               | &check;      | &check;       |
-| BRFLD     | Missing              | Missing               | &check;      | &check;       |
-| BRSA      | Missing              | Missing               | &check;      | &check;       |
-| BRSPD     | Missing              | Missing               | &check;      | &check;       |
-| BSHDAT    | &cross;              | &cross;               | &cross;      | &cross;       |
-| BSMAT     | Missing              | Missing               | &check;      | &check;       |
-| BTUNDA    | Missing              | Missing               | &check;      | &check;       |
-| BUCT      | &check;              | &check;               | &check;      | &check;       |
-| INI       | Missing              | Missing               | &check;      | &check;       |
-| LC        | &check;              | &check;               | &check;      | &check;       |
-| PKG       | &check;              | &check;               | &check;      | &check;       |
-| TOC       | &check;              | &check;               | &check;      | &check;       |
-| TXT       | &check;              | &check;               | &check;      | &check;       |
-| WEBM      | Missing              | Missing               | &cross;      | &cross;       |
+| Format    | Samus Returns (Read) | Samus Returns (Write) | Dread (Read) | Dread (Write) | Samus Returns (Purpose) | Samus Returns (Notes) |
```

Comment:
these new columns are not specific to MSR. I'm also not sure the notes column is all that useful, at least for the MEDS readme. this is better suited for something like https://wiki.axiodl.com/ (but for MSE formats instead of retro)

Reply:
Oh, is there already a wiki for MSE formats?

Reply:
There sadly is not :(

> I'm also not sure the notes column is all that useful, at least for the MEDS readme

I know Henrique has asked in the past explicitly for it to get added. IMO, having a very quick description of what the general format is about (especially while we dont have a wiki yet), helps non-familiar people to know on a quick glance what is actually supported. Telling them "bcmdl and bctex aren't supported, but bmsnd are" is way less helpful for them than "models and textures arent supported, but sounds are".

Reply:
Also because I haven't made it clear before: I do think the "notes" section is useless and should be removed. Having a quick one-liner on "purpose" is more than enough.

Reply:
I'm fairly fine with a quick blurb, though that could just be the purpose. We could have a wiki in this repository for more details on each format.

Reply:
+1 for a wiki in the git repo. Would love to see a main hub for modders to be able to grab pertinent information from.

Reply:
I would like to have a wiki, but IMO the workflow for wikis in git repos is kinda meh.
AFAIK it only allows editing from either owners or everyone (latter is bad because of vandalism, former is bad because admittedly we're all lazy), and major search engines either still dont crawl it or used to not crawl it.
I have a draft wiki locally that i have never pushed anywhere, mostly because i have no clue on most of the file formats and would rather have something filled out before I push it.

Reply:
I would recommend something like MediaWiki, however that requires someone host it on a server, be it local or remote. The real benefit to Github's wiki format is that it doesn't need to be hosted by any one person. But the downsides are as you mentioned, more room for vandalism and upkeep issues. But I can confirm I've found github wikis from googling things before.

Reply:
dunc already posted antidote's mediawiki above, we could probably discuss with him to host a mercury styled one instead.
but i personally dont like mediawiki that much because it also makes submitting changes harder, is harder to get themed (i very much value dark mode), and AFAIK is harder to write content for as you cant just use markdown.

My local-wiki-like-thing is using hugo, which i've always found more comfortable to work with. It also doesnt require someone to host it, as we can just host it off of github pages since it'd be a static website.